raiden/settings.py

```diff
@@ -3,6 +3,7 @@
 
 INITIAL_PORT = 40001
 
+RAIDEN_VERSION = "0.0.6"  # Keep in sync with version in setup.py
```

Comment:
we don't need this. use `raiden.utils.get_system_spec()['raiden']` or this line directly: https://github.com/raiden-network/raiden/blob/master/raiden/utils/__init__.py#L160

Reply:
oh thanks. Will do that now.